### PR TITLE
doc/sample: nit clarifies the vendor caveats

### PR DESCRIPTION
**Pull request by camilamacedo86**
**Description**
doc/sample: nit clarifies the vendor caveats

### CODE REVIEW

**Review by laxmikantbpandhare — APPROVED** [review]
/lgtm

File: hack/generate/samples/internal/go/v3/memcached_with_webhooks.go [hunk 1]
```diff
@@ -440,6 +440,8 @@ func (r *MemcachedReconciler) deploymentForMemcached(m *cachev1alpha1.Memcached)
 							// The memcached image does not use a non-zero numeric user as the default user.
 							// Due to RunAsNonRoot field being set to true, we need to force the user in the
 							// container to a non-zero numeric user. We do this using the RunAsUser field.
+							// However, if you are looking to provide solution for K8s vendors like Openshift
+							// be aware that if you inform this value you can not run under its restricted-v2 SCC.
```
Comment (jmrodri) [inline hack/generate/samples/internal/go/v3/memcached_with_webhooks.go:444]:
```suggestion
							// However, if you are looking to provide solution for K8s vendors like OpenShift
							// be aware that you can not run under its restricted-v2 SCC if you set this value.				 
```

File: website/content/en/docs/best-practices/pod-security-standards.md [hunk 2]
```diff
@@ -88,6 +88,11 @@ dep:= &appsv1.Deployment{
                        "ALL",
                     },
                  },
+                 // The memcached image does not use a non-zero numeric user as the default user.
+                 // Due to RunAsNonRoot field being set to true, we need to force the user in the
+                 // container to a non-zero numeric user. We do this using the RunAsUser field.
+                 // However, if you are looking to provide solution for K8s vendors like Openshift
+                 // be aware that if you inform this value you can not run under its restricted-v2 SCC.
```
Comment (jmrodri) [inline website/content/en/docs/best-practices/pod-security-standards.md:95]:
```suggestion
                 // However, if you are looking to provide solution for K8s vendors like OpenShift
                 // be aware that you can not run under its restricted-v2 SCC if you set this value.
```

File: testdata/go/v3/memcached-operator/controllers/memcached_controller.go [hunk 3]
```diff
@@ -183,6 +183,8 @@ func (r *MemcachedReconciler) deploymentForMemcached(m *cachev1alpha1.Memcached)
 							// The memcached image does not use a non-zero numeric user as the default user.
 							// Due to RunAsNonRoot field being set to true, we need to force the user in the
 							// container to a non-zero numeric user. We do this using the RunAsUser field.
+							// However, if you are looking to provide solution for K8s vendors like Openshift
+							// be aware that if you inform this value you can not run under its restricted-v2 SCC.
```
Comment (jmrodri) [inline testdata/go/v3/memcached-operator/controllers/memcached_controller.go:187]:
```suggestion
							// However, if you are looking to provide solution for K8s vendors like OpenShift
							// be aware that you can not run under its restricted-v2 SCC if you set this value.
```

**Review by jmrodri — DISMISSED** [review]
minor rewording.

**Review by rashmigottipati — APPROVED** [review]
/lgtm

**Pull request merged**